src/Resources/Grapher/CorrelationsResource.php

```diff
@@ -96,4 +73,45 @@ public function delete($id)
         throw new MethodNotFoundException("Method delete() can not be use with correlations");
     }
     #endregion
+
+    /**
+     * @param int    $id
+     * @param string $forClassName
+     *
+     * @return array|Collection
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getById(int $id, string $forClassName)
+    {
+        $params = $this->getQueryParams();
+
+        $type      = CorrelationTypeHelper::getGraphTypeByClass($forClassName);
+        $uriAppend = "{$type}/{$id}";
```

Comment:
`$uriAppend` variable is used only ones

Reply:
This is templated format (all resources functions is written like this)